app/views/producers/_fat.html.haml

```diff
@@ -1,91 +1,92 @@
-.row.active_table_row{"ng-if" => "open()", "ng-click" => "toggle($event)", "ng-class" => "{'open' : open()}"}
-  .columns.small-12.fat.text-center{"ng-show" => "open() && shopfront_loading"}
-    %p.fullwidth
-      = render partial: "components/spinner"
+= cache_with_locale do
```

Comment:
Ah, so the only dynamic content in this file is a translated string, and the cache is valid for as long as the local file doesn't change. Very clever!


For `producers/skinny.html.haml`, it's smaller and there's no dynamic content, so I guess wasn't worth caching?

Reply:
Hmm, but there's no cache key specific for this block, will await your response on `def cache_key_with_locale`.

Reply:
Yes, cache keys in views are already scoped to each partial by default. This is handled under the hood by the default `cache` method, which is `ActionView::Helpers::CacheHelpers#cache`, which is the method we're wrapping in `cache_with_locale` (and passing it some extra arguments so it knows about locales).